index.js

```diff
@@ -369,6 +369,53 @@ module.exports = {
       })
     }
 
+    function excludeMembers(groupId, feedIds, opts = {}, cb) {
+      if (cb === undefined)
+        return promisify(excludeMembers)(groupId, feedIds, opts)
+
+      // TODO: should probably check this against a spec of its own
+      // TODO: should add members tangle to this. should we add opts to publish()? { spec, tangles, keys}
+      publish(
+        {
+          type: 'group/exclude',
+          excludes: feedIds,
+          recps: [groupId],
```

Comment:
This `recps` is the tricky bit and it's the heart of the exclusion. It *cannot* be `groupId`, it has to be the root MF ID of each member minus those being excluded. If there are more than 15, then we need to publish multiple msgs. (And I think we can't use the `publish()` helper)

Reply:
Ignore everthing I said, I'm dumb!